parm/config/gfs/config.aeroanlgenb

```diff
@@ -21,7 +21,7 @@ export aero_diagb_weight=0.9
 export aero_staticb_rescaling_factor=2.0
 export aero_diagb_n_halo=4
 export aero_diagb_n_neighbors=16
-export aero_diagb_smooth_horiz_iter=200
+export aero_diagb_smooth_horiz_iter=0
```

Comment:
We want @CoryMartin-NOAA 's change committed at [3eb4954](https://github.com/CoryMartin-NOAA/global-workflow/commit/3eb495453b9eeda351d41ce658ea2c31b81b1e53).  

See lines  [24-29](https://github.com/CoryMartin-NOAA/global-workflow/blob/3eb495453b9eeda351d41ce658ea2c31b81b1e53/parm/config/gfs/config.aeroanlgenb#L24-#L29).

```
aero_diagb_smooth_horiz_iter=0
export aero_diagb_smooth_vert_iter=0
if [[ "${CASE_ANL}" == "C384" ]]; then
  aero_diagb_smooth_horiz_iter=200
fi
export aero_diagb_smooth_horiz_iter
```

Reply:
I just merged @CoryMartin-NOAA's branch. Thanks @RussTreadon-NOAA